Obsidian/Net/Rcon/RconConnection.cs

```diff
@@ -244,6 +244,13 @@ await SendAsync(new RconPacket(currentEncoding)
 
                         try
                         {
+                            logger.LogInformation("Executing '{Prefix}{Command}'", CommandHandler.DefaultPrefix, packet.PayloadText);
```

Comment:
Debug log message in information scope is bad

Reply:
It's not for debugging purposes, Notchian server does the same. Also, would you rather know if someone got unauthorized access to the server through RCON?